### ISSUE TITLE
[Regression] Call of Duty: Black Ops 3 crashes at a specific loading screen (more details in the post)

### ISSUE THREAD

**Issue by gabriele2000** [doitsujin/dxvk#3681]
I have this weird error with Call of Duty: Black Ops 3

`Assertion failed: !status && "vkDestroyBufferView", file ../dlls/winevulkan/loader_thunks.c, line 3226`

_Originally posted by @gabriele2000 in https://github.com/doitsujin/dxvk/issues/3677#issuecomment-1742146493_

How to reproduce:
- Open the game
- Begin the second mission
- You can skip the cutscene, and the in-game cutscene
- A loading animation will begin, the game will close, the "Assertion Failed" error will appear in wine's log

I've bisected the problem, it happens since a287566c655b7ccc86cc866cd2bf8a075a4d4708

```
a287566c655b7ccc86cc866cd2bf8a075a4d4708 is the first bad commit
commit a287566c655b7ccc86cc866cd2bf8a075a4d4708
Author: Philip Rebohle <philip.rebohle@tu-dortmund.de>
Date:   Fri Jun 16 16:07:48 2023 +0200

    [dxgi] Implement frame statistics based on IDXGIVkSwapChain1

 src/dxgi/dxgi_swapchain.cpp | 123 ++++++++++++++++++++++++++++++--------------
 src/dxgi/dxgi_swapchain.h   |   5 +-
 2 files changed, 89 insertions(+), 39 deletions(-)
```

**Comment by Blisto91** [comment]
Are you sure of that bisect? Thought you said it happened on multiple dxvk versions.

**Comment by gabriele2000** [comment]
> Are you sure of that bisect? Thought you said it happened on multiple dxvk versions.

I've started a new bisect and now I'm sure that this is the offending commit

**Comment by Blisto91** [comment]
@gabriele2000 What is the status of this?

**Comment by gabriele2000** [comment]
> @gabriele2000 What is the status of this?

Well, at some point it stopped doing it, I think it's a bug in the game because it randomly triggered, but that happened a year ago

**Comment by Blisto91** [comment]
@gabriele2000 I assume that this is still working to this day?

**Comment by gabriele2000** [comment]
> [@gabriele2000](https://github.com/gabriele2000) I assume that this is still working to this day?

Yeah, plus DXVK wasn't the real issue, more than an almost-random bug from the game.
